### PR TITLE
Fixes for operator init command

### PR DESCRIPTION
**Pull request by ostromart**
https://github.com/istio/istio/issues/20579

### CODE REVIEW

**Review by morvencao — APPROVED** [review]

**Review by elfinhe — APPROVED** [review]

**Review by bianpengyuan — APPROVED** [review]

File: mixer/adapter/opa/config/adapter.opa.config.pb.html [hunk 1]
```diff
@@ -24,8 +24,8 @@ <h2 id="Params">Params</h2>
   - |+
     package mixerauthz
     policy = [
-      &lbrace;
```
Comment (howardjohn) [inline mixer/adapter/opa/config/adapter.opa.config.pb.html]: the common files and build tools image is mismatched causing this @jasonwzm

**Pull request merged**